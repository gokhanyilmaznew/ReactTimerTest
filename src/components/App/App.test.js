import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

describe("App component snapshot", () => {
    test("should create snapshot", () => {
    
        expect().toMatchSnapshot();
    });
});