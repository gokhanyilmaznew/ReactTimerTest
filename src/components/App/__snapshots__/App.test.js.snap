// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component snapshot should create snapshot 1`] = `undefined`;

exports[`TimerButton component snapshot should create snapshot 1`] = `undefined`;
